src/electron/go_vpn_tunnel.ts

```diff
@@ -136,10 +136,12 @@ export class GoVpnTunnel implements VpnTunnel {
   }
 
   private suspendListener() {
-    // Swap out the current listener, restart once the system resumes.
+    // Windows: when the system suspends, tun2socks terminates due to the TAP device getting closed.
+    // Preemptively stop tun2socks without notifying its exit.
     this.tun2socks.onExit = () => {
```

Comment:
Remove this. I think changing this handler is confusing. The output in the stop() method can differentiate stop from crash. Perhaps you just have a log output here saying it's suspending.

Reply:
I agree that swapping the listener is not the best. The issue with that approach is that the process gets restarted immediately after receiving the suspend event, at which point the TAP device is not available, so restarting fails.

Reply:
Done.